### PR TITLE
Add Typing

### PR DESCRIPTION
**Pull request by rnetser**

### CODE REVIEW

File: webhook_server_container/libs/github_api.py [hunk 1]
```diff
@@ -94,22 +94,22 @@ def __init__(self, hook_data):
...
+        self.container_repository_password: str
+        self.container_repository: str
+        self.dockerfile: str
+        self.container_tag: str
+        self.container_build_args: List
+        self.container_command_args: List
+        self.repository_full_name: str
+        self.github_app_id: str
+        self.container_release: bool
+        self.can_be_merged_required_labels: Any
```
Comment (myakove) [inline webhook_server_container/libs/github_api.py:109]: List[str]

**Review by myakove — CHANGES_REQUESTED** [review]
Can you please add mypy to pre-commit?

**Comment by myakove** [comment]
!

**Review by myakove — COMMENTED** [review]
!

**Review by rnetser — COMMENTED** [review]
sasdasddsa

**Review by rnetser — COMMENTED** [review]
zczczxzczxczxc

**Review by myakove — APPROVED** [review]

**Pull request merged**